--- ssh_server_fork.c	2018-08-10 17:06:03.000000000 +0800
+++ ssh_server_fork_patch.c	2018-10-19 13:44:07.000000000 +0800
@@ -531,7 +531,7 @@
     ssh_event_add_session(event, session);
 
     n = 0;
-    while (sdata.authenticated == 0 || sdata.channel == NULL) {
+    while (sdata.channel == NULL) {
         /* If the user has used up all attempts, or if he hasn't been able to
          * authenticate in 10 seconds (n * 100ms), disconnect. */
         if (sdata.auth_attempts >= 3 || n >= 100) {
